File: src/java/fib.java
class fib {
    static int fib(Integer n) {
        if (n == 0 || n == 1) {
            return n;
        }
        else {
            return fib(n - 1) + fib(n - 2);
        }
    }
    public static void main (String[] args) {
        System.out.println(fib(15));
    }
}

File: src/fortran/fib.f90
program fibonacci
  integer :: n, result

  ! Set the value of n
  n = 15 

  ! Calculate Fibonacci
  result = fibonacci_recursive(n)

  ! Display the result
  print *, result

contains

  recursive function fibonacci_recursive(m) result(fib)
    integer, intent(in) :: m
    integer :: fib

    if (m <= 0) then
      fib = 0
    else if (m == 1) then
      fib = 1
    else
      fib = fibonacci_recursive(m - 1) + fibonacci_recursive(m - 2)
    end if
  end function fibonacci_recursive

end program fibonacci